packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -172,9 +172,15 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
   };
 
   private _onToggleCodeClick = () => {
-    this.setState({
-      isCodeVisible: !this.state.isCodeVisible
-    });
+    if (this.props.isCodeVisible) {
+      this.props.onClick!('');
```

Comment:
can we please avoid the use of `!` postfix operator, or if deemed to make a lot of sense, include a JSDoc explaining why? As more and more `!` creeps into the code (and the moment you see one, others will be tempted to add more), the harder it gets to reason over correctness and type safety significantly degrades...

Reply:
Added extra conditional to check and avoid '!' operator

---
In reply to: [302383654](https://github.com/OfficeDev/office-ui-fabric-react/pull/9769#discussion_r302383654) [](ancestors = 302383654)